src/test/java/org/jenkinsci/plugins/badge/extensions/SpecialValueParameterResolverExtensionTest.java

```diff
@@ -2,6 +2,8 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
```

Comment:
Thanks for the addition, but this source file already uses a Mockito.mock() call without needing a static import.  Let's keep it consistent within the source file.  Likewise, the source file uses Hamcrest assertions rather than assertEquals from JUnit.  This is a good place for the static import of nullValue().
```suggestion
import static org.hamcrest.Matchers.nullValue;
```